CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Master
+- IntOrString is now object thus it can be both int or string.
```

Comment:
`it can store either int or string` right?

Reply:
Yes. The downside is it can be stored anything (to that point, any other field can be anything as python is not type safe) but server will complain about anything other than int or string.